index.css

```diff
@@ -0,0 +1,267 @@
+*{
+    margin: 0; 
+    padding: 0;
+} 
+a{
+    text-decoration: none;
+    color: #696969;
+}
+a:visited{
+    text-decoration: none;
+    color: #696969;
+}
+
+/* for travel page */
+h1{
+   position: relative; 
+   z-index: 1;
+   top: 100px;
+   left: 200px;
+}
```

Comment:
if you give the `h1` a class name or id it will be better.

Reply:
I didn't because it is the only "h1" in the page. 
But I should be consistent.